src/Weasel.Postgresql/NpgsqlTypeMapping.cs

```diff
@@ -103,6 +104,7 @@ public class NpgsqlTypeMapper
         {NpgsqlDbType.Path, new NpgsqlTypeMapping(NpgsqlDbType.Path, DbType.Object, "path", typeof(NpgsqlPath))},
         {NpgsqlDbType.Point, new NpgsqlTypeMapping(NpgsqlDbType.Point, DbType.Object, "point", typeof(NpgsqlPoint))},
         {NpgsqlDbType.Polygon, new NpgsqlTypeMapping(NpgsqlDbType.Polygon, DbType.Object, "polygon", typeof(NpgsqlPolygon))},
+        {NpgsqlDbType.Geometry, new NpgsqlTypeMapping(NpgsqlDbType.Geometry, DbType.Object, "geometry", typeof(Geometry))},
 
         // LTree types
```

Comment:
To support geography, I think this would do the trick. Geometry and  Geography uses the same type in the NetTopologySuite. Would not icluding `typeof(Geometry)`  cause issues here?
`{NpgsqlDbType.Geography, new NpgsqlTypeMapping(NpgsqlDbType.Geography, DbType.Object, "geography")},`